brd-android/trade/src/main/java/com/fabriik/trade/ui/TradeUrlBuilder.kt

```diff
@@ -7,6 +7,7 @@ object TradeUrlBuilder {
     fun build(supportedCurrencies: List<String>): Uri {
         val merchantId = "NGVQYXnFp13iKtj1"
         val currencies = supportedCurrencies.joinToString(",")
+        val refId = "m3RudWCT8RWVKSrp"
```

Comment:
🤔 I do not understand the change. It looks like the hardcoded value was moved from line 22 to line 10.

Reply:
Please check this change: https://github.com/fabriik/wallet-android/commit/52b012f47fa54f8c3a07d2621457e0a9d0eb0af9

Lenart committed this change to `mit-android` branch by mistake and then opened this PR with this small change.

Reply:
Please check this change: https://github.com/fabriik/wallet-android/commit/52b012f47fa54f8c3a07d2621457e0a9d0eb0af9

Lenart committed this change to `mit-android` branch by mistake and then opened this PR with this small change.

Reply:
Yeah  it's strange I know, basically I included line 22 in my change from ticket but I accidentally pushed it directly to mit-android. Because it was a one liner I didn't want to make a big mess with reverting it and then adding another commit so i just refactored it a little bit and then put it on separate branch so that u and Rok can check it. Since the base change was pushed directly to mit-android it now looks like i only changed the line order :/

Reply:
So to sum up I made mistake with pushing it directly to mit-android but since it's one liner and I assumed everything was fine I left it be. To be 100% sure I created another PR so that u and Rok can double check. Thinking about it now I should probably just revert the direct commit and made one proper PR :)